exercises/practice/robot-name/robot_name_test.go

```diff
@@ -63,8 +63,8 @@ func TestResetName(t *testing.T) {
 	}
 }
 
-// Test 1000 names are unique - this should run reasonably quickly even with a sub-optimal solution
-// (e.g. pick a random name, then pick a new name if it's been seen before)
+// Test 1000 names are unique - this should run reasonably quickly even with a solution that is not optimized
+// with respect to running time. (e.g. pick a random name, then pick a new name if it's been seen before)
```

Comment:
Thanks for the improvement. I am not a native speaker but according to the dictonary it is called "run time" not "running time". Same applies below.

Reply:
In my field, "runtime" refers to the general state of the system during a program's execution: "runtime libraries", "runtime memory allocation". I deliberately used "running time" to emphasize the fact that we are talking about the classic CPU vs memory tradeoff here since "running time"'s meaning is unambiguously tied to the time it takes for a program to execute (whether that time is measured in wall time or CPU time is immaterial here).

Reply:
I'd personally suggest changing:
> that is not optimised with respect to running time.

to
> that is not particularly fast.

and getting rid of the stuff in brackets altogether. I don't think the example is needed and just gives a clue to the student about what is optimised and what is not.

Reply:
I have been curious about "runtime" and how different fields use this, or change this.  What is your field?

Reply:
Or, rather, which field?  As I see you are involved in more than one.

Reply:
AFAIK "runtime" refers to the base environment/library/interpreter your code runs on, while "run time" is the actual time **when** your code is running, not its duration.

I'd probably stick to speed-related terms on this one.